Sources/TSCBasic/CMakeLists.txt

```diff
@@ -6,6 +6,11 @@
 # See http://swift.org/LICENSE.txt for license information
 # See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 
+if(CMAKE_VERSION VERSION_LESS 3.16 AND CMAKE_SYSTEM_NAME STREQUAL Windows)
```

Comment:
these changes should go in swiftpm right now and then brought here using the Utilities/import script.